### DISCUSSION TITLE
Standards supported by E-ID

### DISCUSSION THREAD

**Discussion by damienbod** [e-id-admin/open-source-community#6] [Q&A]
At present **BBS+** , **DIDComm V2** and **SIOP** seem to be the standards which most SSI solutions converge on. If the E-ID solution should be used outside of the BIT and county solutions, these would need to be supported. Without these standards supported, it will be really expensive to support E-ID when producing new software solutions. The E-ID solution should not create a technology monopoly for the solution it chooses.

Question 

How will the E-ID solution  support these standards for SSI solutions and make it easy to adopt the VC from E-ID?

**Comment by christianheimann** [comment]
To enrich this thread: may you @damienbod share your pros and cons of the mentionned standards?

**Comment by damienbod** [comment]
The standards **BBS+**, **DIDComm V2** and **SIOP** looks like what is required if you need to allow a majority of solutions to work with the E-ID wallet, if SSI ever becomes mainstream. Half the SSI solutions will support SIOP in the future (guessing) and the other half will support DIDComm V2. At present no standards are supported by all and most solutions use legacy standards or vendor specific implementations which do not work with other vendors. Inter-op is a bit of a mess in the SSI space at present.

At present most systems, services, demos use legacy standards which will fade out. 

BBS+ looks like the VC standard definition which is gaining most support. 

SIOP provides a standard authentication solution of the user where the agents using DIDComm V2 require the vendor specific wallet solutions. 

I believe the E-ID should not build a system based on legacy standards or standards which have no future. A lot of providers are moving towards these 3 standards as well as other standards. 

A disadvantage of not using these standards (or a widespread of standards which allows the majority of SSI service providers), is a technology lockdown and will result in high costs to onboard or use the E-ID solution in third party solutions as it is vendor specific and not supported by at least 50% of the SSI providers. Implementing a complete SSI solution to use the E-ID would not make sense (too complex), so third party solution providers are reliant on using a solution which will work with the E-ID.

Greetings Damien
